app/main.py

```
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split(" ")
+    if len(parts) == 3 and parts[0] == "cp" and parts[1] != parts[2]:
+        cp, source_file_name, destination_file_name = parts
+        try:
+            with (open(source_file_name, "r") as file_in,
+                  open(destination_file_name, "w") as file_out):
+                file_out.write(file_in.read())
+        except FileNotFoundError:
+            pass
```

While it's good that you've narrowed the exception from a general `Exception` to the specific `FileNotFoundError`, silencing it with `pass` is still problematic. If the source file doesn't exist, the function should raise an error to inform the user, not fail silently. Consider removing the `try...except` block and letting the `FileNotFoundError` propagate naturally.